docs/_documentations/remotedeploy-projects-eclipse.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: docs
+title: Eclipse
+description: Creating and importing projects
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing cloud connection, remote deployment of Codewind
+duration: 5 minutes
+permalink: remotedeploy-projects-eclipse
+type: document
+---
+
+# Creating and importing projects
+
+Now that you have configured Codewind to be used remotely, you are ready to create a project that you can develop locally but build and run remotely.  
+
+**Note:** If you have projects that currently exist on your local computer that you would like to continue to develop remotely, follow [Import an existing local project to the cloud](#import-an-existing-local-project-to-the-cloud).
+
+## Create a new project
+
+Go to the **Codewind Explorer** view, right-click your remote connection, and select **Create New Project**. 
+
+![Creating new remote project](./images/remotedeploy-projects-eclipse/eclipse_create-new-project.png){:width="380"}
+
+Select the template type and enter a project name. Your new project is built and after a few seconds begins running in the cloud.
+
+## Import an existing local project to the cloud
+
+Copy an existing local project, for example `myFirstNodeProject`, over to the remote Codewind deployment:
+
+1\. Right-click your remote connection and select **Add Existing Project**:
+
+![Add existing project](./images/remotedeploy-projects-eclipse/eclipse_add-project.png){:width="366"}
+
+2\. Click **Select a project from the workspace** and select your project: 
+
+![Add to Codewind](./images/remotedeploy-projects-eclipse/eclipse_select-project.png){:width="654"}
+
+Codewind prompts you to confirm the project type, in this case, identified as `Node.js`. Confirm the project type and click **Finish**:
+
+![Confirm Project Type](./images/remotedeploy-projects-eclipse/eclipse_confirm-project.png){:width="654"}
+
+If you add a project to the remote connection that is already deployed on the local connection, the **Project Already Deployed** window appears: 
+
+![Project Already Deployed](./images/remotedeploy-projects-eclipse/eclipse_project-deployed.png){:width="524"}
```

Comment:
Not a huge deal, but it might be good to keep the capitalization consistent even in the alt text for images and change them to sentence case.
- Line 39: Confirm project type
- Line 43: Project already deployed
Etc.